### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-de/content_actorconditions.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#270]
```
GERMAN Translation

If you find any mistakes, tell me please thanks
```

Original issue reported on code.google.com by `cngunner...@googlemail.com` on 3 Nov 2011 at 9:28

Attachments:
- [content_actorconditions.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-270/comment-0/content_actorconditions.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately I found a bigger mistake:
There is already a German Translation of the content_actorconditions.xml.

On the Translation Howto:
http://code.google.com/p/andors-trail/wiki/Translation_Howto

there is a link to a Validator:
http://andors-trail.googlecode.com/svn/trunk/AndorsTrailEdit/translations.html

There you could have found out about this.

Or have a look here:
http://code.google.com/p/andors-trail/source/browse/#svn%2Ftrunk%2FAndorsTrail%2
Fres%2Fvalues-de

Also the translation is not so good at some places. Will post about differences 
of the 2 translations here in a few minutes:
http://andorstrail.com/viewtopic.php?f=6&t=329

I do not want to criticize you. I just want to prevent you from work that had 
been done already. I am glad to see some progress on the German translation.
```

Original comment by `SamuelPl...@gmail.com` on 3 Nov 2011 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
Remark: You changed the identifier "poison_weak" to "weak_poison".
This is not possible. The first column must remain constant.
```

Original comment by `SamuelPl...@gmail.com` on 3 Nov 2011 at 10:46

**Comment by GoogleCodeExporter** [comment]
```
Here is the post about the differences of the 2 concurring translations:
http://andorstrail.com/viewtopic.php?p=6948#p6948

We need some Feedback there.
```

Original comment by `SamuelPl...@gmail.com` on 3 Nov 2011 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
Thank you for your comments, I will try to translate other file, because I want 
100% german-speaking game immediately.

```

Original comment by `cngunner...@googlemail.com` on 4 Nov 2011 at 8:41

**Comment by GoogleCodeExporter** [comment]
```
Nice to hear that. I think I do not have the time for translating itself, but I 
will help reviewing the translations.

I made an overview what needs a translation here:
http://andorstrail.com/viewtopic.php?p=6950#p6950
```

Original comment by `SamuelPl...@gmail.com` on 4 Nov 2011 at 8:49

**Comment by GoogleCodeExporter** [comment]
```
According to the forum these changes should be done for 
"values-de/content_actorconditions.xml":
See attached patchfile.
```

Original comment by `SamuelPl...@gmail.com` on 8 Nov 2011 at 12:28

Attachments:
- [German Active Condition update.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-270/comment-6/German Active Condition update.patch)

**Comment by GoogleCodeExporter** [comment]
```
I translated the rest of the "content_itemlist.xml".
Patchfile is attached.
```

Original comment by `SamuelPl...@gmail.com` on 18 Nov 2011 at 10:02

Attachments:
- [German Itemlist update.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-270/comment-7/German Itemlist update.patch)

**Comment by GoogleCodeExporter** [comment]
```
I translated the "content_monsterlist.xml".
Patchfile is attached.
```

Original comment by `SamuelPl...@gmail.com` on 19 Nov 2011 at 1:02

Attachments:
- [German Monsterlist.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-270/comment-8/German Monsterlist.patch)

**Comment by GoogleCodeExporter** [comment]
```
Issue 269 has been merged into this issue.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:45

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:46
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
